.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,6 +16,7 @@ Fixes #
 - [ ] Write detailed docstrings for all functions/methods.
 - [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
 - [ ] If adding new functionality, add an example to docstrings or tutorials.
+- [ ] Use underscores (not hyphens) in names of Python files and directories.
```

Comment:
Should this also say that underscores should be used in function and variable names?

Reply:
> Should this also say that underscores should be used in function and variable names?

In Python, hyphens are not allowed in function and variable names. So we have to use underscores when function and variable names are long.

Reply:
Did not know that! Thanks!